.github/workflows/misc.yml

```
@@ -51,3 +51,9 @@ jobs:
         echo "::error::Replace !errors.Is(err, target) with testify equivalents"
         exit 1
 
+    - name: Check for zero width unicode characters
+      run: |
+        grep -r -n -I --color=always --exclude-dir=.git -P "[\x{200B}\x{200C}\x{200D}\x{FEFF}\x{2060}]" . || exit 0
```

Searched for other invisible (zero width characters) and found some characters we can add.
```suggestion
        grep -r -n -I --color=always --exclude-dir=.git -P "[\x{00A0}\x{2000}-\x{200F}\x{2028}-\x{202F}\x{205F}-\x{206F}\x{FEFF}\x{3000}]" .
```